src/main/java/seedu/contax/ui/onboarding/OnboardingWindow.java

```diff
@@ -23,6 +29,9 @@
     private OnboardingCommandBox commandBox;
     private OnboardingInstruction instructionLabel;
     private UniquePersonList persons;
```

Comment:
You should not maintain a UniquePersonList, but a dummy `Model`.
```suggestion
    private Mode model;
```

Reply:
Updated accordingly thanks